lib/cc/engine/analyzable_files.rb

```diff
@@ -19,7 +18,7 @@ def all
       def fetch_files(paths)
         paths.map do |path|
           if path =~ %r{/$}
-            Dir.glob("#{path}/**/*.coffee")
+            Dir.glob("#{path}**/*.coffee")
```

Comment:
I didn't test it, but since we only seemed to `cd` in `CoffeelineResults` I assumed this wouldn't work properly. I can pull out the `cd` changes if that's not the case and we want to extract that into a separate PR.